website/content/api-docs/secret/pki.mdx

```diff
@@ -16,1178 +16,1109 @@ update your API calls accordingly.
-- [Sign Verbatim](#sign-verbatim)
-- [Tidy](#tidy)
-- [Tidy Status](#tidy-status)
+- [Notice About New Multi-Issuer Functionality](#notice-about-new-multi-issuer-functionality)
+- [Accessing Authority Information](#accessing-authority-information)
+  - [List Issuers](#list-issuers)
+  - [Read Issuer Certificate](#read-issuer-certificate)
+  - [Read Default Issuer Certificate Chain](#read-default-issuer-certificate-chain)
+  - [Read Issuer CRL](#read-issuer-crl)
+  - [List Certificates](#list-certificates)
+  - [Read Certificate](#read-certificate)
+  - [Read Raw Certificate](#read-raw-certificate)
+- [Issuing Certificates](#issuing-certificates)
+  - [List Roles](#list-roles)
+  - [Generate Certificate and Key](#generate-certificate-and-key)
+  - [Sign Certificate](#sign-certificate)
+  - [Sign Intermediate](#sign-intermediate)
+  - [Sign Self-Issued](#sign-self-issued)
+  - [Sign Verbatim](#sign-verbatim)
+  - [Revoke Certificate](#revoke-certificate)
+- [Managing Keys and Issuers](#managing-keys-and-issuers)
+  - [List Issuers](#list-issuers)
+  - [List Keys](#list-keys)
+  - [Generate Root](#generate-root)
+  - [Generate Intermediate](#generate-intermediate)
+  - [Import CA Certificates and Keys](#import-ca-certificates-and-keys)
+  - [Read Issuer](#read-issuer)
+  - [Update Issuer](#update-issuer)
+  - [Delete Issuer](#delete-issuer)
+  - [Read Key](#read-key)
+  - [Update Key](#update-key)
+  - [Delete Key](#delete-key)
+  - [Delete All Issuers and Keys](#delete-all-issuers-and-keys)
+- [Managing Authority Information](#managing-authority-information)
+  - [List Roles](#list-roles)
+  - [Create/Update Role](#create-update-role)
+  - [Read Role](#read-role)
+  - [Delete Role](#delete-role)
+  - [Read CRL Configuration](#read-crl-configuration)
+  - [Set CRL Configuration](#set-crl-configuration)
+  - [Read URLs](#read-urls)
+  - [Set URLs](#set-urls)
+  - [Read CRL](#read-crl)
+  - [Rotate CRLs](#rotate-crls)
+  - [Tidy](#tidy)
+  - [Tidy Status](#tidy-status)
+- [Notes on Personas](#notes-on-personas)
+- [Cluster Scalability](#cluster-scalability)
 - [Managed Key](#managed-keys) (Enterprise Only)
+- [Vault CLI with DER/PEM responses](#vault-cli-with-der-pem-responses)
 
-## Read CA Certificate
+## Notice About New Multi-Issuer Functionality
 
-This endpoint retrieves the CA certificate _in raw DER-encoded form_. This is a
-bare endpoint that does not return a standard Vault data structure and cannot
-be read by the Vault CLI; use `/pki/cert` for that. If `/pem` is added to the
-endpoint, the CA certificate is returned in PEM format.
+Vault now allows a single PKI mount to have multiple Certificate Authority
+(CA) certificates ("issuers") in a single mount, for the purpose of
+facilitating rotation. All issuers within a single mount are treated as a
+single authority, meaning that Certificate Revocation List (CRL) configuration
+and other authority URLs are common to all issuers and serial numbers will be
```

Comment:
nit: parsing this sentence is hard since (all issuers and serial numbers) parses together to me: can we write `all issuers, moreover serial numbers...`